airbyte-db/lib/src/main/resources/configs_database/schema2.sql

```diff
@@ -0,0 +1,218 @@
+CREATE
+    TYPE "source_type" AS enum(
+        'api',
+        'file',
+        'database',
+        'custom'
+    );
+
+CREATE
+    TYPE "actor_type" AS enum(
+        'source',
+        'destination'
+    );
+
+CREATE
+    TYPE "operator_type" AS enum(
+        'normalization',
+        'dbt'
+    );
+
+CREATE
+    TYPE "namespace_definition_type" AS enum(
+        'source',
+        'destination',
+        'customformat'
+    );
+
+CREATE
+    TYPE "status_type" AS enum(
+        'active',
+        'inactive',
+        'deprecated'
+    );
+
+CREATE
+    TABLE
+        IF NOT EXISTS "workspace"(
+            id uuid NOT NULL,
+            customer_id uuid NULL,
+            name VARCHAR(256) NOT NULL,
+            slug VARCHAR(256) NOT NULL,
+            email VARCHAR(256) NULL,
+            initial_setup_complete BOOLEAN NOT NULL,
+            anonymous_data_collection BOOLEAN NULL,
+            news BOOLEAN NULL,
+            security_updates BOOLEAN NULL,
+            display_setup_wizard BOOLEAN NULL,
+            tombstone BOOLEAN NOT NULL DEFAULT FALSE,
+            notifications jsonb NULL,
+            first_completed_sync BOOLEAN NULL,
```

Comment:
`first_sync_complete` (or `first_sync_completed`, if you go for this one then the column above should be `initial_setup_completed`)